src/pretix/base/signals.py

```diff
@@ -122,3 +122,9 @@ def send(self, sender: Event, **named) -> List[Tuple[Callable, Any]]:
 idempotent, i.e. it should not make a difference if this is sent out more often
 than expected.
 """
+
+register_global_settings = django.dispatch.Signal()
+"""
+All plugins that are installed may send fields for the global settings form, as
+an OrderedDict of (setting name, form field).
```

Comment:
I believe the correct API would be "as an object behaving like a dict", but since an OrderedDict is the most sensible thing to do, we can leave this like this.